website/docs/getting-started/switching-podman-machine-default-connection.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 10
+title: Setting Podman machine default connection
+description: How to set Podman machine default connection.
+---
+
+## Setting Podman machine default connection
+
+Each Podman machine is exposing two connections:
+
+- rootless
+- rootful
+
+Podman has one default connection.
+
+Podman Desktop, and other tools, such as Kind, connect to the default connection.
```

Comment:
should it be Kind or `kind` ?

Reply:
We already settled Kind, see: https://podman-desktop.io/docs/kubernetes/kind